web/blog/tenzir-v4.19/index.md

```diff
@@ -0,0 +1,221 @@
+  blocklists, helping network owners to protect their users from Dridex and
+  Emotet/Heodo.
+```
+
+Every package _must_ have a unique identifier. We recommend setting the package
+name, description and author, and we also recommend setting an author and a
+package icon where possible.
+
+Packages _may_ define inputs. This, alongside the author and package icons, will
+become relevant in an upcoming release that adds interactive browsing and
+installation of ready-made packages on [app.tenzir.com](https://app.tenzir.com).
+For now, we'll just leave this out:
+
+```yaml {0} title="feodo/package.yaml [2/5]"
+config:
+  inputs: {}
+```
+
+Packages _may_ define any number of contexts. For our Feodo Abuse Blocklist
+package we'll define a context named `feodo` as a [Lookup
+Table](/contexts/lookup-table). We recommend writing a description for every
+context.
+
+```yaml {0} title="feodo/package.yaml [3/5]"
+contexts:
+  feodo:
+    type: lookup-table
+    description: |
+      A lookup table that contains the elements of the feodo IP blocklist.
+```
+
+Packages _may_ define any number of pipelines. These pipelines get automatically
+started when the package is installed. For our example, let's add a pipeline
+that ensures that our `feodo` context is continuously updated:
+
+```yaml {0} title="feodo/package.yaml [4/5]"
+pipelines:
+  update-context:
+    name: Update Feodo Context
+    description: |
+      Periodically refresh the Feodo lookup-table context.
+    definition: |
+      every 1 hour from https://feodotracker.abuse.ch/downloads/ipblocklist_aggressive.csv read csv --allow-comments
+      | context update feodo --key dst_ip
+```
+
+The format for pipelines matches the format for [Pipelines as
+Code][pipelines-as-code].
+
+Lastly, we recommend adding snippets to your package that show just how to use
+it:
+
+```yaml {0} title="feodo/package.yaml [5/5]"
+snippets:
+  - name: Match historical and live data against the `feodo` context
+    description: |
+      Find persisted events that have an IP address matching the `feodo`
+      context.
+    definition: |
+      lookup feodo --field :ip
+  - name: Visualize successful lookups with the `feodo` context in the last week
+    description: |
+      Creates a stacked area chart that shows the number of hourly hits of
+      pipelines using the `lookup` operator with the `feodo` context.
+    definition: |
+      metrics lookup
+      | where context == "feodo"
+      | where timestamp > 7d ago
+      | summarize retro_hits=sum(retro.hits), live_hits=sum(live.hits) by timestamp resolution 1h
+      | sort timestamp
+      | chart area --position stacked
+```
+
+That's it! Our own package, all done and wrapped up.
+
+:::tip Want to dive deeper?
+We've prepared some reading material:
+- Take a look at the [`package` operator's documentation](/operators/package).
+- Read the user guide on [installing a package](/user-guides/install-a-package).
```

Comment:
@lava I think we still need to write this—we can probably take the same example that we used for this blog post, and then in addition also explain inputs and packages deployed as code.